src/sagas/session.js

```diff
@@ -136,7 +151,9 @@ export function* listBuckets(
     }
     const responses = yield call([client, client.batch], batch => {
       for (const { id } of data) {
-        batch.bucket(id).listCollections();
+        batch
+          .bucket(id)
+          .listCollections({ limit: sidebarMaxListedCollections + 1 });
```

Comment:
Yeah wasn't sure... but that allows us to always have one more item than what's in `sidebarMaxListedCollections`, so in the sidebar we can check if the number of items is strictly greater than `sidebarMaxListedCollections` to know if there are more entries on the server, and so we can render a link to paginated list of collections. That's admittedly convoluted, but works great :)